scm-ui/ui-webapp/src/users/components/UserConverter.tsx

```diff
@@ -22,24 +22,18 @@
  * SOFTWARE.
  */
 import React, { FC, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import classNames from "classnames";
+import { User } from "@scm-manager/ui-types";
+import { useConvertToExternal, useConvertToInternal } from "@scm-manager/ui-api";
 import {
   Button,
   ErrorNotification,
   Level,
   Modal,
   PasswordConfirmation,
-  SubmitButton
+  SubmitButton,
 } from "@scm-manager/ui-components";
-import { useTranslation } from "react-i18next";
-import { User } from "@scm-manager/ui-types";
-import styled from "styled-components";
-import { useConvertToExternal, useConvertToInternal } from "@scm-manager/ui-api";
-
-const ExternalDescription = styled.div`
-  display: flex;
-  align-items: center;
-  font-weight: 400;
```

Comment:
Why did this have a 400 font-weight ? Was it removed on purpose ?

Reply:
`400` is still normal thickness.